runtime/common/src/constants.rs

```diff
@@ -301,3 +302,10 @@ parameter_types! {
 	/// 42 is the prefix for the Frequency Network on Rococo
 	pub const Ss58Prefix: u16 = prod_or_testnet_or_local!(90, 42, 42);
 }
+
+// --- Capacity Pallet ---
+pub type CapacityMinimumStakingAmount = ConstU128<{ currency::EXISTENTIAL_DEPOSIT }>;
+pub type CapacityMaxUnlockingChunks = ConstU32<4>;
+pub type CapacityMaxEpochLength = ConstU32<100>;
+pub type CapacityUnstakingThawPeriod = ConstU16<2>;
```

Comment:
nit: what is the unit of these periods? some description would be nice

Reply:
The units are documented in the `capacity/src/lib.rs` file for the pallet constant that these values are setting in runtime. For example, UnstakingThawPeriod is the number of epochs. 

I would not be opposed to duplicating some code comments here so long as we maintain them for consistency with comments to the capacity pallet constants.